src/components/Confirm/style.css

```diff
@@ -18,9 +18,16 @@
   padding: 0;
 }
 
-.fileViewer{
-  width: 92%;
-  height: 290px;
-  max-height: 70vh;
-  border: 0;
+:global {
+  #pdfIcon {
```

Comment:
it shouldn't be a global. It should be a class called `.pdfIcon`, which you can then refer to on your JS component like this:
```
<div class=`{style.pdfIcon}`></div>
```

This will namespace your selector, which is great and essential to reduce selector collision with the host app.